pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -245,6 +245,8 @@
   front_join_us_learn_more: "Learn more"
 
   signup_closed: 'Registration for the Hour of Code is now closed. Please check again in October to sign up for the next Hour of Code.'
+  signup_registration_not_required: "You don't need to register to participate! The Hour of Code activities are available year round. <a href='%{hoc_activities_url}'>Try an activity</a> or read our <a href='%{howto_guide_url}'>How-to guide</a> to plan an event for your class."
+  signup_registration_closed: "And every year in December, your class can join millions of students around the world celebrating Computer Science Education Week with the Hour of Code. We'll open registration for the annual celebration in October."
```

Comment:
Is there a time when we don't show these together?  Having both `_not_required` and `_closed` feels like they are shown at different times, for slightly different reasons, rather than next to each other.

Reply:
Sorry, somehow missed this comment before merging. I separated them because it felt like the first sentence could be used on its own and I want to try and isolate translations as much as possible with complete thoughts in case we do want to use just one or the other in the future.